apps/nextjs-website/src/_contents/webinars.ts

```diff
@@ -58,7 +58,7 @@ export const webinars: readonly Webinar[] = [
     title: 'Esplorando App IO: I messaggi a contenuto remoto',
     description:
       'Tutto sulla nuova funzionalità di remotizzazione dei contenuti',
-    playerSrc: 'https://vimeo.com/event/4176802/embed',
+    playerSrc: 'https://player.vimeo.com/video/930599737',
```

Comment:
**question**: is there a reason why old webinars were provided by `vimeo.com/event/...` and this one should be reachable to `player.vimeo.com/video/...`?

Reply:
Because of a Vimeo limit, which in some cases does not allow the live URL to be reused for video on demand.
From a user's perspective, nothing changes and the third-level domain will soon be necessary to publish other videos on demand